app/controllers/users_controller.rb

```diff
@@ -858,6 +858,26 @@ def password_reset_show
     end
   end
 
+  def remove_password
+    RateLimiter.new(nil, "remove-password-hr-#{request.remote_ip}", 6, 1.hour).performed!
+    RateLimiter.new(nil, "remove-password-min-#{request.remote_ip}", 3, 1.hour).performed!
+
+    user = fetch_user_from_params
+    guardian.ensure_can_edit!(user)
```

Comment:
I guess a question for product, whether staff should be able to remove people's passwords. I'm not sure there's a use case for this. 🤔